docs/api/wasm/extensions.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: docu
+title: Extensions
+selected: Client APIs
+---
+
+Default [extensions](../../extensions/overview) enabled in DuckDB-wasm are HTTPFS and Parquet.
```

Comment:
I think also json and fts (= full text search) are packaged by default.

Reply:
Actually, correcting myself, Parquet, FTS are alwasy packaged by default, while JSON and Excel are build-time opt-in. See https://github.com/duckdb/duckdb-wasm/blob/968ba722350d9be3988d7f44ac328c0ef0c8423a/lib/CMakeLists.txt#L238, I believe JSON and Excel

HTTPFS it's not really an extension, but a different re-implementation of it that comes bundled by default.

Reply:
File changed